app/models/piece.rb

```diff
@@ -58,6 +60,12 @@ def obstructed?(new_x, new_y)
 
   private
 
+  def increment_move
```

Comment:
Rubocop doesn't like me using the increment! method because it says it skips validations. What's a better way to write this?